automan/automation.py

```diff
@@ -1070,5 +1073,10 @@ def _setup_argparse(self):
             dest='update_remote', default=False,
             help='Update remote worker machines.'
         )
+        parser.add_argument(
+            '-del', '--delete-outputs', action='store_true',
```

Comment:
I do not like the `-del`, instead use a single character for the single-dash case.  Also is `--delete-remote` what this means?  Please think of a more suitable name that describes what is being done.

Reply:
I can't think of a suitable single character argument, So can we just keep `--delete-outputs`.